oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/segment/SegmentRecord.java

```diff
@@ -91,7 +90,7 @@
     private byte[] dataBinary;
     @Setter
     @Getter
-    @Column(columnName = TAGS)
+    @Column(columnName = TAGS, indexOnly = true)
```

Comment:
If the tags are only indexed, how can they be displayed in the UI, or we don't display these ever?

Reply:
All segment info include `tags` are stored in `dataBinary` already.